### PR TITLE
add expenses faq

### PR DESCRIPTION
**Pull request by piamancini**

### CODE REVIEW

File: frontend/src/containers/Faq.js [hunk 1]
```diff
@@ -75,6 +75,24 @@ export class Faq extends Component {
           <h3>How do I make a donation?</h3>
           <p>You rock! We are delighted you are asking this question. Go to the Collective’s Public Page (e.g. https://opencollective.com/wwcodeatl), choose your recurring donation amount and hit the Back Us or Donate button, fill out the form et voilà! Now you are part of that collective, your community will appreciate it.</p>
           
+          <h3>How do I submit an expense?</h3>
+          <p>Go to your collective page to submit an expense. You will need to upload a valid image for it to get reimbursed. Don’t forget to include your paypal email if it’s different from your user’s email.</p>
```
Comment (asood123) [inline frontend/src/containers/Faq.js]: s/user email/user's email

File: frontend/src/containers/Faq.js [hunk 2]
```diff
@@ -75,6 +75,24 @@ export class Faq extends Component {
...
+          <p>There are two ways of using your Collective’s money:
```
Comment (asood123) [inline frontend/src/containers/Faq.js]: s/Collective/collective

File: frontend/src/containers/Faq.js [hunk 3]
```diff
@@ -75,6 +75,24 @@ export class Faq extends Component {
...
+            <ul>
+              <li>Submit a receipt for a service you paid for (online services, meals, plane tickets).</li>
```
Comment (asood123) [inline frontend/src/containers/Faq.js]: Submit a receipt for something you paid for like online services, meals, plane tickets, etc.

File: frontend/src/containers/Faq.js [hunk 4]
```diff
@@ -75,6 +75,24 @@ export class Faq extends Component {
...
+              <li>Submit an invoice for your work. The invoice must include your current address and be made to Open Collective Inc. (or to the host if you are hosted by another organisation) with reference to your collective.</li>
+             </ul>
+          </p>
+          <p>Expenses will be paid once a week on Fridays. Please contact support@opencollective.com if you have any questions.</p>
```
Comment (asood123) [inline frontend/src/containers/Faq.js]: can we make support@ a mailto link?

File: frontend/src/containers/Faq.js [hunk 5]
```diff
@@ -75,6 +75,24 @@ export class Faq extends Component {
...
+          <h3>How do I submit an expense?</h3>
+          <p>Go to your collective page to submit an expense. You will need to upload a valid image for it to get reimbursed. Don’t forget to include your paypal email if it’s different from your user’s email.</p>
+          <p>There are two ways of using your Collective’s money:
+            <ul>
+              <li>Submit a receipt for a service you paid for (online services, meals, plane tickets).</li>
+              <li>Submit an invoice for your work. The invoice must include your current address and be made to Open Collective Inc. (or to the host if you are hosted by another organisation) with reference to your collective.</li>
+             </ul>
+          </p>
+          <p>Expenses will be paid once a week on Fridays. Please contact support@opencollective.com if you have any questions.</p> 
+          <p>Open Collective’s details are:</p>
```
Comment (asood123) [inline frontend/src/containers/Faq.js]: Open Collective details:

**Comment by piamancini** [comment]
@asood123 changes done

**Pull request merged**